### ISSUE TITLE
Can't load swirl

### ISSUE THREAD

**Issue by AbidAliShaikh** [swirldev/swirl#233]
I have installed swirl, but when I try to load it gives me error; please someone help me!

library(swirl)
Error in inDL(x, as.logical(local), as.logical(now), ...) : 
  unable to load shared object 'C:/Documents and Settings/Jon/My Documents/R/win-library/3.1/yaml/libs/i386/yaml.dll':
  LoadLibrary failure:  %1 is not a valid Win32 application.

Error: package or namespace load failed for ‘swirl’

**Comment by Bobbietto** [comment]
Hey guys! I have a similar problem, can anyone tell me why this happens?

![image](https://cloud.githubusercontent.com/assets/13000793/8282947/56f13778-18f4-11e5-8988-f9d82f5ed171.png)

**Comment by Bobbietto** [comment]
> library("swirl")
Error in loadNamespace(j <- i[[1L]], c(lib.loc, .libPaths()), versionCheck = vI[[j]]) : 
  there is no package called ‘crayon’
Error: package or namespace load failed for ‘swirl’

as you can see, I keep on having the same issue

**Comment by ncarchedi** [comment]
Try install.packages("swirl", dependencies = TRUE). If that doesn't work, just install the package it says you're missing using the same command.

**Comment by npefley** [comment]
I am unable to install 'crayon'. It failed when ran install.packages("swirl"), when I ran it with dependencies = TRUE, when I ran install.packages("crayon"), and when I ran that with dependencies = TRUE.  I'm using a mac running OS X Mavericks if it matters. Is it just me, or does 'crayon' not exist?

Thanks!

Sample error message:

> install.packages("crayon")

  There is a binary version available but the source version is later:
       binary source needs_compilation
crayon  1.2.1  1.3.0             FALSE

installing the source package ‘crayon’

trying URL 'http://cran.cnr.Berkeley.edu/src/contrib/crayon_1.3.0.tar.gz'
Error in download.file(url, destfile, method, mode = "wb", ...) : 
  cannot open URL 'http://cran.cnr.Berkeley.edu/src/contrib/crayon_1.3.0.tar.gz'
In addition: Warning message:
In download.file(url, destfile, method, mode = "wb", ...) :
  cannot open: HTTP status was '404 Not Found'
Warning in download.packages(pkgs, destdir = tmpd, available = available,  :
  download of package ‘crayon’ failed
